renovate/defaults.json5

```diff
@@ -78,37 +78,59 @@ Validate this file before commiting with (from repository root):
    ***** Manager-specific configuration options *****
    **************************************************/
 
-  "regexManagers": [
+  "customManagers": [
+    // Track the latest CI VM images by tag on the containers/automation_images
+    // repo.  Propose updates when newer tag available compared to what is
+    // referenced in a repo's .cirrus.yml file.
     {
+      "customType": "regex",
       "fileMatch": "^.cirrus.yml$",
       // Expected veresion format: c<automation_images IMG_SFX value>
       // For example `c20230120t152650z-f37f36u2204`
       "matchStrings": ["c(?<currentValue>20\\d{6}t\\d{6}z-\\w+)"],
       "depNameTemplate": "containers/automation_images",
       "datasourceTemplate": "github-tags",
       "versioningTemplate": "loose",
-      "autoReplaceStringTemplate": "c{{{newVersion}}}",
+      "autoReplaceStringTemplate": "c{{{newVersion}}}"
     },
+
+    // For skopeo and podman, manage the golangci-lint version as
+    // referenced in their Makefile.
     {
+      "customType": "regex",
       "fileMatch": "^Makefile$",
       // make ignores whitespace around the value, make renovate do the same.
-      "matchStrings": ["GOLANGCI_LINT_VERSION\\s+:=\\s+(?<currentValue>.+)\\s*"],
+      "matchStrings": [
+        "GOLANGCI_LINT_VERSION\\s+:=\\s+(?<currentValue>.+)\\s*"
+      ],
       "depNameTemplate": "golangci/golangci-lint",
       "datasourceTemplate": "github-releases",
       "versioningTemplate": "semver-coerced",
       // Podman's installer script will puke if there's a 'v' prefix, as represented
       // in upstream golangci/golangci-lint releases.
-      "extractVersionTemplate": "v(?<version>.+)",
+      "extractVersionTemplate": "v(?<version>.+)"
     }
   ],
 
+  /*************************************************
+  ***** Language-specific configuration options ****
+  **************************************************/
+
+  // *****  ATTENTION  WARNING  CAUTION  DANGER  ***** //
+  // Go versions 1.21 and later will AUTO-UPDATE based on module
+  // requirements.  ref: https://go.dev/doc/toolchain  Fedora 40+
+  // disable this feature but that doesn't affect other distros,
+  // nor the behavior of the public renovate service.  The only
```

Comment:
Maybe add something like
> Unlike most Go dependencies, we don’t want the Go toolchain version to be updated to the latest possible version, because we need the codebases to be buildable using all currently-maintained version of Fedora.

somewhere in here, just to have that recorded for future maintainers?

Reply:
Or is it really
> Force Go 1.20 here to avoid the `go toolchain` behavior altogether
(regardless of the Fedora need to stay on Go 1.20)?

Perhaps both?

Reply:
Ahh that's a great suggestion for more context.  Thanks!